packages/ui/src/components/Banners/LearnWeb3Banner.tsx

```diff
@@ -10,119 +10,129 @@ export interface HomePageBannerProps {
+                <p className="pl-6">
+                  <span className="text-[#F97583]">require</span>
+                  (uintVote&nbsp;<span className="text-[#F97583]">&lt;&nbsp;</span>
+                  <span className="text-[#9ECBFF]">3, &quot;must be less than 3&quot;</span>);
+                </p>
+              </div>
+            </div>
+            <div className="line">
+              <div>8</div>
+              <div>
+                <p className="pl-6 text-white">
+                  <span className="text-[#FF39F7]">Vote&nbsp;</span>
+                  <span className="text-[#e65a9e]">vote</span> =&nbsp;
+                  <span className="text-[#B392F0]">Vote</span>(uintVote);
+                </p>
+                <p className="pl-3 text-white">...</p>
+              </div>
+            </div>
           </div>
-        </div>
-        <div className="line">
-          <div>2</div>
-          <div className="text-white">
-            <p>
-              )
-              <span className="text-[#F97583]">
-                public <span className="text-[#BE2D61]">nonReentrant onlyDelegate</span>
-              </span>
-              &#123;
-            </p>
-          </div>
-        </div>
-        <div className="line">
-          <div>3</div>
-          <div className="text-white">
-            <p className="pl-6">
-              <span className="text-[#FF39F7]">address&nbsp;</span>
-              <span className="text-[#e65a9e]">member Address</span> =
-            </p>
-            <p className="pl-12">
-              memberAddress[<span className="text-[#79B8FF]">msg</span>.sender];
-            </p>
-          </div>
-        </div>
-        <div className="line">
-          <div>4</div>
-          <div>
-            <p className="pl-6 text-white md:whitespace-nowrap">
-              <span className="text-[#FF39F7]">Member&nbsp;</span>
-              <span className="text-[#e65a9e]">storage member</span> = members[memberAddress];
-            </p>
-          </div>
-        </div>
-        <div className="line">
-          <div>5</div>
-          <div className="text-white">
-            <p className="pl-6">
-              <span className="text-[#F97583]">require</span>
-              (proposalIndex <span className="text-[#F97583]">&lt;&nbsp;</span>
-              proposalQueue.length,
-            </p>
-            <p className="pl-12">
-              <span className="text-[#9ECBFF]">&quot;proposal does not exist&quot;</span>);
-            </p>
+          <div className="p-10">
+            <CardHeader className="flex-row justify-center">
+              <CardTitle className="title">LEARN WEB3</CardTitle>
+            </CardHeader>
+            <CardContent className=" p-1 pb-12 text-center text-sm font-light text-white md:mb-[40px] md:pb-1 md:text-[17px]">
+              <p>
+                Developer DAO offers learning <br /> resources to help you build with web3 <br />
```

Comment:
In the figma file the text is:

`DeveloperDAO Academy offers learning resources to help you learn how to build web3 technologies. `

![image](https://github.com/user-attachments/assets/8b3cf035-1eef-4379-b753-45960d8a3a8f)

Can you update it? 
_Academy_ after _DeveloperDAO_ is missing and _DeveloperDAO_ is alltogether.

Reply:
Changed the text from `Developer DAO` to `DeveloperDAO`